Makefile

```diff
@@ -55,3 +55,9 @@ uninstall-pre-commit:
 
 print-python-env:
 	@./tools/print_env.sh
+
+sign-off:
+	echo "git interpret-trailers --if-exists doNothing \c" >> .git/hooks/commit-msg
+	echo '--trailer "Signed-off-by: $$(git config user.name) <$$(git config user.email)>" \c' >> .git/hooks/commit-msg
+	echo '--in-place "$$1"' >> .git/hooks/commit-msg
```

Comment:
This adds the following line to the bottom of `.git/hooks/commit-msg`:
```
git interpret-trailers --if-exists doNothing --trailer "Signed-off-by: $(git config user.name) <$(git config user.email)>"--in-place "$1"
```
It took a truly horrifying amount of time to figure out all the right escape characters and things to get this working 😱